docs/esterni/manuali/manuale_acquirente.tex

```diff
@@ -104,8 +104,7 @@ \subsection{Visualizzazione acquisti}
   \end{itemize}
 \end{itemize}
 \subsection{Gestione acquisti}
-Per ciascun acquisto l'acquirente ha 14 giorni per annullare il pagamento e dunque essere rimborsato con la valuta stabile DAI. Nel caso l'utente voglia annullare il pagamento dovrà selezionare l'opzione "Refund transaction" (Fig. 4 riquadro 2) e dunque ricevere il rimborso.
-\end{itemize}
+Dopo 14 giorni dalla data di acquisto, se i fondi non sono stati sbloccati dall'acquirente attraverso la scannerizzazione del codice QR, avverrà il rimborso in DAI dell'esatto importo pagato. Nel caso l'acquirente voglia annullare il pagamento manualmente, potrà selezionare l'opzione "Refund transaction" (Fig. 4 riquadro 2) e dunque ricevere il rimborso.
```

Comment:
Enfatizzerei che il refund è usabile solo dopo i 14 giorni se non avviene in automatico (in caso di problemi è il piano B).